src/main/java/org/opensearch/agent/tools/PPLTool.java

```diff
@@ -373,4 +405,20 @@ private String parseOutput(String llmOutput, String indexName) {
         return ppl;
     }
 
+    private String loadDefaultPrompt(PPLModelType pplModelType) {
+        try (InputStream searchResponseIns = PPLTool.class.getResourceAsStream("PPLDefaultPrompt.json");) {
+            if (searchResponseIns != null) {
+                String defaultPromptContent = new String(searchResponseIns.readAllBytes(), StandardCharsets.UTF_8);
+                Map<String, String> defaultPromptDict = gson.fromJson(defaultPromptContent, Map.class);
+                return defaultPromptDict.get(pplModelType.toString());
```

Comment:
What if pplModelType is empty string ("")? This happens when user gives neither prompt and model_type

Reply:
The default value for model type is claude and will use default prompt for claude if both fields are none.

Reply:
got it